### PR TITLE
Document Java production profile

### PR DESCRIPTION
**Pull request by beckermarc**

### CODE REVIEW

File: java/developing-applications/configuring.md [hunk 1]
```diff
@@ -16,21 +16,38 @@ uacp: Used as link target from Help Portal at https://help.sap.com/products/BTP/
...
-CAP Java applications can fully leverage [Spring Boot's](../spring-boot-integration) capabilities for [Externalized Configuration](https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config). 
+This section describes how to configure applications.
+CAP Java applications can fully leverage [Spring Boot's](../spring-boot-integration) capabilities for [Externalized Configuration](https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config).
 This enables you to define multiple **configuration profiles** for different scenarios, like local development and cloud deployment.
 
 For a first introduction, have a look at our [sample application](https://github.com/sap-samples/cloud-cap-samples-java) and the [configuration profiles](https://github.com/SAP-samples/cloud-cap-samples-java/blob/master/srv/src/main/resources/application.yaml) we added there.
 
 Now, that you're familiar with how to configure your application, start to create your own application configuration. See the full list of [CDS properties](properties) as a reference.
 
+### Production Profile
```
Comment (BraunMatthias) [inline java/developing-applications/configuring.md:27]:
```suggestion
### Production Profile { #production-profile }
```

**Review by BraunMatthias — APPROVED** [review]

File: java/developing-applications/configuring.md [hunk 2]
```diff
@@ -16,21 +16,38 @@ uacp: Used as link target from Help Portal at https://help.sap.com/products/BTP/
...
 Now, that you're familiar with how to configure your application, start to create your own application configuration. See the full list of [CDS properties](properties) as a reference.
 
+### Production Profile { #production-profile }
+
+When running your application in production it makes sense to strictly disable some development-oriented features.
+The production profile configures a set of selected property defaults, recommended for production deployments, at once.
+To specify the production profile set `cds.environment.production.profile` to a Spring profile used in production deployments.
+
+::: tip
+It is recommended to set `cds.environment.production.profile` to `cloud`. The Java Buildpacks set the `cloud` profile for applications by default.
```
[the remark this answers is not in the document]
Comment (beckermarc) [reply java/developing-applications/configuring.md:34]: The CDS property should follow what you configure in the buildpack. If you already today use a different profile for your deployed application in production you should adjust `cds.environment.production.profile` to use that.

[the remark this answers is not in the document]
Comment (renejeglinsky) [reply java/developing-applications/configuring.md:34]: I couldn't find how to configure the buildpack wrt the profile. Should we add that information? Maybe in line 38ff?

[the remark this answers is not in the document]
Comment (beckermarc) [reply java/developing-applications/configuring.md:34]: It is not really a buildpack configuration. There are a lot of ways in Spring Boot to configure the active profile. The most common is setting the environment variable `SPRING_PROFILES_ACTIVE`. I will add that to the docs.

[the remark this answers is not in the document]
Comment (renejeglinsky) [reply java/developing-applications/configuring.md:34]: Yes, thanks a lot! :)

**Review by renejeglinsky — APPROVED** [review]

**Pull request merged**